package.json

```diff
@@ -22,10 +22,10 @@
     "preview": "vite preview",
     "start": "run-p start:server start:watch:ts start:watch:css",
     "start:server": "vite",
-    "start:watch:ts": "tsc --watch",
+    "start:watch:ts": "npm run build:src:js -- --watch",
```

Comment:
There is an additional `--` after build:src:js?

Reply:
Yes, this is the way it works: https://docs.npmjs.com/cli/v9/commands/npm-run-script

You split arguments to `npm` and arguments you want to pass down to the script, `tsc` in this case with double dash

Reply:
Got it, thanks for explaining